src/Modifiers/CoreModifiers.php

```diff
@@ -2398,12 +2406,14 @@ public function embedUrl($url)
             if (Str::contains($url, '?t=')) {
                 $url = str_replace('?t=', '?start=', $url);
             }
+        }
 
-            return $url;
+        if (Str::contains($url, 'youtube.com/watch?v=')) {
+            $url = str_replace('watch?v=', 'embed/', $url);
         }
 
-        if (Str::contains($url, 'vimeo')) {
-            return str_replace('/vimeo.com', '/player.vimeo.com/video', $url);
+        if (Str::contains($url, 'youtube.com')) {
+            $url = str_replace('youtube.com', 'youtube-nocookie.com', $url);
```

Comment:
If you are doing the no cookie stuff for `youtube.com`, could you not also do it for `youtu.be`?

Reply:
I think it's doing it for both cases 🙂

`youtu.be` is changed to `youtube.com` which then later is changed to `yotube-nocookie.com`.

This was the way with the least levels of indentation and repetition i could come up with 😅

Reply:
Ah okay... I've not tested it out, just looked over the new code.

Reply:
I could write a couple unit tests if that helps...